skimage/transform/_warps.py

```diff
@@ -66,10 +66,10 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
     anti_aliasing : bool, optional
         Whether to apply a Gaussian filter to smooth the image prior
         to down-scaling. It is crucial to filter when down-sampling
```

Comment:
```suggestion
        to down-sizing. It is crucial to filter when down-sizing
```
Sorry for requesting this change affecting lines which do not strictly belong to the PR... But I find it way too confusing to juggle between 'down-sizing,' 'down-sampling,' and 'down-sampling.' Let's stick with 'down-sizing,' since the function is called 'resize' and as the PR description says: "Added clarification on kwarg anti_aliasing description: it is applied by
default for non bool type inputs when _downsizing_."